README.md

```diff
@@ -25,13 +25,14 @@ global manipulation. Our goal with **lab** is to keep the execution engine as si
 - `-C`, `--colors` - enables or disables color output. Defaults to console capabilities.
 - `-d`, `--dry` - dry run. Skips all tests. Use with `-v` to generate a test catalog. Defaults to `false`.
 - `-D`, `--debug` - print the stack during a domain error event.
+- `--debug-brk` - break on the first line of application code.
```

Comment:
If it's the same as Node's built in, then we shouldn't document it. I'm not sure it's the behavior we'd want either.